### ISSUE TITLE
Mac Bug: Cannot try/test 1.2.0 Permissions bug and the program launching after with "the program is not open anymore"

### ISSUE THREAD

**Issue by godzfire** [rustdesk/rustdesk#3936]
### Bug Description

This is a continuation of the bug: [Windows to Mac OS: Command-Tab key combo (Windows-Tab) to switch between open apps constantly scrolls to end of list](https://github.com/rustdesk/rustdesk/issues/3863#top) #3863

Ruskdesk closed the ticket saying to try the nightly build. However I cannot try 1.2.0 because of the [Bug] Screen Recording permission invalid on macOS #3261 (https://github.com/rustdesk/rustdesk/issues/3261), where Rustdesk will keep erroring out and saying "the program is not open anymore".

That ticket was closed by Ruskdesk last week without a resolution even when someone asked.

It was referenced in [Unable to establish a connection to a Macbook using Rustdesk despite having permission for screen recording](https://github.com/rustdesk/rustdesk/issues/2797#top) #2797, which was closed with notes: "close to favor https://github.com/rustdesk/rustdesk/issues/3261". So we're back to this, [[Bug] Screen Recording permission invalid on macOS](https://github.com/rustdesk/rustdesk/issues/3261#top) #3261.

Long story short, there was this loop of issues that were closed and either referencing another ticket which was also closed and referenced back, or closed without a solution.

How can we test out nightly builds if we are experiencing the permissions bug and the "is not open anymore" bug which doesn't allow us to run the program?

### How to Reproduce

Try to run nightly build on Mac.

### Expected Behavior

Able to set permissions and run the program

### Operating system(s) on local side and remote side

Mac

### RustDesk Version(s) on local side and remote side

1.2.0 nightly

### Screenshots

NA

**Comment by rustdesk** [comment]
just reopened https://github.com/rustdesk/rustdesk/issues/3261
